packages/cdktf-cli/bin/cmds/handlers.ts

```diff
@@ -197,6 +197,13 @@ export async function init(argv: any) {
   await displayVersionMessage();
   await checkEnvironment();
 
+  console.log(argv);
+  if (argv.fromTerraformProject === "") {
+    throw Errors.Usage(
+      "--from-terraform-project requires a path to an existing Terraform project to be set, e.g. --from-terraform-project=../my-tf-codebase"
```

Comment:
should we add a short info, that using the current directory does not work because that is where cdktf init will initialize the new project?